v1.1/array.md

```diff
@@ -0,0 +1,64 @@
+---
+title: ARRAYS
+summary: The ARRAYS data type stores 
+toc: false
+---
+
+The `ARRAY` data type creates one-dimensional, 1-indexed, homogenous arrays of any non-array [data type](data-types.html).
+
+<div id="toc"></div>
+
+{{site.data.alerts.callout_info}} CockroachDB does not support nested arrays, indexing on arrays, and ordering by arrays.{{site.data.alerts.end}}
+
+## Syntax
+A column of data type `ARRAY` can be created in the following ways:
```

Comment:
Reworded as suggested. 
Also, I don't think the content in the [value expressions doc](https://www.cockroachlabs.com/docs/dev/sql-expressions.html#array-constructors) has been affected by this PR. @justinj Can you verify please?